numpy/f2py/tests/test_crackfortran.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import division, absolute_import, print_function
+
+import pytest
+
+import numpy as np
+from numpy.testing import assert_array_equal
+from . import util
+
+
+class TestNoSpace(util.F2PyTest):
```

Comment:
nit: Docstring might be handy here to explain what `NoSpace` means

Reply:
adding